fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/tree/FabricTreeRegistry.java

```diff
@@ -76,6 +47,14 @@
 	 * Creates a new instance of a {@link FoliagePlacerType}, registers and returns it.<br>
 	 * Uses {@link FoliagePlacerTypeInvoker} under the hood.
 	 *
+	 * <p>
+	 *     Example:<br>
+	 *
+	 *     <code>
+	 *         public static final FoliagePlacerType<MyFoliagePlacer> MY_FOLIAGE_PLACER = FabricTreeRegistry.registerFoliagePlacerType(new Identifier("tutorial", "my_foliage_placer"), MyFoliagePlacer.CODEC);
```

Comment:
Hmm, I don't think the generic diamonds work with generated html. Replace `<code></code>` with `{@code}` instead to avoid this issue. Same for other usages.

Reply:
Okay, thanks! I'll do that tomorrow since it's 10 PM in my timezone and I kinda have to sleep.

Reply:
Done!